src/Ocelot/Security/SecurityPolicyExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Ocelot.Security;
+
+public static class SecurityPolicyExtensions
+{
+    public static string GetClientIpAddress(this HttpContext httpContext, bool tryUseXForwardHeader = true)
+    {
+        if (httpContext == null)
+        {
+            return null;
+        }
+
+        string ip = null;
```

Comment:
what ip? What about using more concrete variable name like clientIpAddress?

Reply:
I believe this is minor issue of readability, because the IP var is single variable, so we don't have multiple IP vars.
Also, the type of IP can be easily recognized from the method name which is `GetClientIpAddress`.
Your requirement comes from some procedural languages where a function can have multiple variable containing the word and it is better to match the names of the function and returning result.

Is this major or minor issue on your opinion?

Reply:
minor ofc, just the naming finese